fabfile.py

```diff
@@ -44,17 +44,6 @@ def server_deploy():
         )
 
 
-def reset_migrations():
-    """Development only - remove and recreate all migration"""
-    for app_name in base.LOCAL_APPS:
-        local('rm -f {}/migrations/ -r'.format(app_name))
-
-    for app_name in base.LOCAL_APPS:
-        manage('makemigrations {}'.format(app_name))
-
-    local('git add . --all')
```

Comment:
Guilty as charged. It had its place and time - when it was me and @bmarika only working in the same room, shouting changes across the table.

That time is gone.